styles.css

```diff
@@ -57,38 +68,109 @@ div ul li {
     color: #F3A774;
     border-bottom: 4px solid #F3A774;
 }
-form {
+.tabs {
+    display: none;
+    border-radius: 8px;
+    overflow: hidden;
+}
+#tab {
+    position: absolute;
+    opacity: 0;
+    z-index: -1;
+}
+.tab {
+    width: 100%;
+    overflow: hidden;
+}
+.tab-label {
+    display: -webkit-box;      /* OLD - iOS 6-, Safari 3.1-6 */
+    display: -moz-box;         /* OLD - Firefox 19- (buggy but mostly works) */
+    display: -ms-flexbox;      /* TWEENER - IE 10 */
+    display: -webkit-flex;
+    display: flex;
+    justify-content: space-between;
+    -webkit-justify-content: space-between;     /*iOS*/
+    -ms-flex-pack: justify;
+    padding: 1em;
+    font-weight: bold;
+    cursor: pointer;
+}
+.tab-label:hover {
+    cursor: pointer;
+}
+.tab-label::after {
+    content: "\276F";
+    width: 1em;
+    height: 1em;
+    text-align: center;
+    transition: all .35s;
+}
+.tab-content {
+    max-height: 0;
+    padding: 0 1em;
+    color: #2c3e50;
+    background: white;
+    transition: all .35s;
+}
+/*.tabs input:checked + .tab-label {*/
+/*    background: #1a252f;*/
+/*}*/
+.tabs input:checked + .tab-label::after {
+    -webkit-transform: rotate(90deg);
+    transform: rotate(90deg);
+}
+.tabs input:checked ~ .tab-content {
+    max-height: 100vh;
+    padding: 1em;
+}
+form.big-form, .tabs form {
```

Comment:
'Add new item' and 'submit' boxes touch the side of the screen

Reply:
ok done! thanks!